libs/blocks/aside/aside.css

```diff
@@ -600,10 +884,22 @@
     object-fit: cover;
   }
 
-  .aside.split .split-image img {
+  .aside.split .split-image .play-container,
+  .aside.split .split-image img,
+  .aside.split .split-image video {
     max-width: 1396px;
   }
 
+  .aside.split .desktop-wide img,
+  .aside.split .desktop-wide video {
+    aspect-ratio: var(--aspect-ratio-wide);
+  }
+  
+  .aside.split .desktop-standard img,
+  .aside.split .desktop-standard video {
+    aspect-ratio: var(--aspect-ratio-standard);
+  }
```

Comment:
I see a `square` [option for mobile](https://github.com/adobecom/milo/pull/979/files#diff-172ebed29740bfd3c497f52bd6e25f9eb65802501dafd514cb208bba9f12d0d0R178). No square option for desktop either?

Reply:
@rgclayton As per the design specs shared by the consonant team the square option is available for mobile view only hence we have not added the option for tablet or desktop.